src/Compilers/CSharp/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,6 +1,10 @@
 *REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentSyntax.Update(Microsoft.CodeAnalysis.CSharp.Syntax.NameColonSyntax nameColon, Microsoft.CodeAnalysis.SyntaxToken refOrOutKeyword, Microsoft.CodeAnalysis.CSharp.Syntax.ExpressionSyntax expression) -> Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentSyntax
 *REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentSyntax.WithRefKindKeyword(Microsoft.CodeAnalysis.SyntaxToken refOrOutKeyword) -> Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentSyntax
+*REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.CrefParameterSyntax.RefOrOutKeyword.get -> Microsoft.CodeAnalysis.SyntaxToken
```

Comment:
Has this already shipped?  If so, isn't this a breaking change?

Reply:
@jasonmalinowski did we update the API files when we shipped 15.5?

Reply:
Will keep the old APIs. Looking into why the tool is behaving this way.

---
In reply to: [159702342](https://github.com/dotnet/roslyn/pull/24023#discussion_r159702342) [](ancestors = 159702342)